src/main/java/com/google/sps/data/BusinessesService.java

```diff
@@ -108,4 +129,104 @@ private void addListingToBusinesses(PlacesSearchResult place) {
     allBusinesses.add(new Listing(name, formattedAddress, 
           placeLocation, rating, photos, types));
   }
-}
\ No newline at end of file
+
+  public void checkNumberOfLocationsOfBusiness() {
+    GeoApiContext context = 
+      new GeoApiContext.Builder().apiKey(KEY).build();
+    numberOfSmallBusinesses = 0;
+    Iterator<Listing> businesses =  allBusinesses.iterator();
+    while(businesses.hasNext()
```

Comment:
nit: Add a space above this line to make this more readable.

Thanks for sticking through this.